docs/build/html/_sources/hacking.rst.txt

```diff
@@ -0,0 +1,19 @@
+Hacking
+===================================
+
+Have you have found a bug in the operator or do you have an additional feature? That's great! Here are some tips.
+
+
+Running your own images.
+
+If you have edited the code, you would like to install it in your own cluster, you will first need an account on an image host like dockerhub. Once you have this, you can build, push, and install the image using the **IMG** environment variable
+::
+  export IMG=your-image-repo/ipfs-operator:version
+  make docker-build
+  make docker-push
+  make install
```

Comment:
What is the bundle image for and should it be built etc when testing?

What about `make catalog-build`, `make catalog push`... when should we use those?

Reply:
@hsanjuan make bundle is used once we get the operator ready for operatorhub etc. make bundle will generate a "bundle" of configuration files based on various files used by the operator. Then we ship the bundle items to operatorhub via PR and a new version will publish